examples/with-emotion/package.json

```diff
@@ -7,13 +7,13 @@
     "start": "next start"
   },
   "dependencies": {
-    "babel-plugin-emotion": "^7.1.0",
-    "emotion": "^7.2.0",
-    "emotion-server": "^7.2.0",
+    "babel-plugin-emotion": "8.0.4",
```

Comment:
These should use `^` so we don't get locked into a minor version number.

Reply:
I was actually wondering that, but haven't seen hydrate in the medium post (even though you pointed out that react-emotion exports all of emotion and I've seen it in source) and didn't want to risk it for the off chance that it *should* be imported directly from emotion for some reason :D